frontend/src/views/ProductCreateEdit.js

```diff
@@ -1,13 +1,59 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 
 import ViewWrappers from 'components/ViewWrappers/ViewWrappers';
 import ProductForm from 'components/ProductForm/ProductForm';
+import ProductList from 'components/ProductList/ProductList';
+import useApi from '../shared/api';
 
+const ProductCreateEdit = ({ match }) => {
 
-const FirstProduct = ({ match }) => (
-  <ViewWrappers.View container withPadding>
-    <ProductForm id={match.params.id} editId={match.params.editId}/>
-  </ViewWrappers.View>
-)
+  const [products, setProducts] = useState([]);
+  const { API } = useApi();
+  const id = match.params.id;
+  const editId = match.params.editId;
 
-export default FirstProduct
+  useEffect(() => {
+    const getProducts = async () => {
+      const response = await API.productList.get(id);
+      setProducts(response.data);
+    };
+    getProducts();
+  },[id, API]);
+
+  const productOverview =
+  <ViewWrappers.View className="PhotoSelect" container renderFn={isBigScreen => (
+    <>
+        <ViewWrappers.ViewSplitter withPadding>
+          <ProductList edit={true} products={products}/>
+        </ViewWrappers.ViewSplitter>
+
+      {isBigScreen ?
+        <ViewWrappers.ViewSplitter withPadding>
+            <ProductForm id={match.params.id} editId={match.params.editId}/>
+        </ViewWrappers.ViewSplitter> : null }
+    </>
+  )} />
+
+  const editView =
+  <ViewWrappers.View className="PhotoSelect" container renderFn={isBigScreen => (
```

Comment:
THis still has the old classname. Also it looks like you have the padding added twice.